src/main/webapp/WEB-INF/jsp/submitFeedback.jsp

```diff
@@ -20,91 +20,147 @@
 
+<link href="<c:url value="/css/feedback.css"/>" type="text/css" rel="stylesheet"/>
+
 <script src="<rs:resourceURL value="/rs/jquery/1.11.0/jquery-1.11.0.min.js"/>" type="text/javascript"></script>
+<script src="/ResourceServingWebapp/rs/jquery-migrate/1.2.1/jquery-migrate-1.2.1.min.js" type="text/javascript"></script>
 <script src="<c:url value="/js/twitlimit-0.2.0.compressed.js"/>" type="text/javascript"></script>
 
-<link href="<c:url value="/css/feedback.css"/>" type="text/css" rel="stylesheet"/>
-
 <c:set var="n"><portlet:namespace/></c:set>
 
 <portlet:actionURL var="postUrl"/>
 
-<h1><spring:message code="feedback.form.question"/></h1>
+<div class="feedback-portlet awesome-bootstrap-checkbox bootstrap-styles">
+<h3 style="font-weight: bold;"><spring:message code="feedback.form.question"/></h3>
 
-<form:form action="${postUrl}" modelAttribute="submitFeedbackForm">       
+<form:form action="${postUrl}" modelAttribute="submitFeedbackForm">
     <spring:bind path="prefs.*">
-    	<c:if test="${status.error}">
-    	<div id="${n}error-message" class="error-message portlet-msg-error portlet-msg error" role="alert" style="display:none">
-   	 	<p><form:errors path="feedback"/></p>
-		</div>
-	    </c:if>
-	</spring:bind>
-	<br>
-	<p id="${n}answer">
-      	<form:radiobutton id="yes" path="like" value="YES"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.yes"/></label>
-      	<form:radiobutton id="no" path="like" value="NO"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.no"/></label>
-      	<form:radiobutton id="maybe" path="like" value="MAYBE"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.maybe"/></label>
-    </p>
-
- 	<p> 		
- 		<label class="portlet-form-field-label"><spring:message code="feedback.form.suggestion"/></label>
- 		<textarea id="${n}feedback" path="feedback" name="feedback" rows="${feedbackRows}" style="width:${feedbackWidth}"></textarea> 		
- 	</p>
- 	<div id="${n}limit" style="margin-bottom: 12px;"></div>
-
- 	<p>
-        <form:checkbox path="anonymous" value="true"/>
-		<label class="portlet-form-field-label"><spring:message code="feedback.form.anonymous"/></label>
-	</p>
-
- 	<input id="${n}useragentstring" type="hidden" name="useragent"/>
- 	<input id="${n}feedbacktabname" type="hidden" name="tabname"/>
-
-    <p>
-       <button type="submit" id="submit" class="feedback-submit-button" disabled="disabled"><spring:message code="feedback.form.submit"/></button>
-    </p>
+        <c:if test="${status.error}">
+        <div id="${n}error-message" class="alert alert-danger" role="alert" style="display:none">
+            <p><form:errors path="feedback"/></p>
+        </div>
+        </c:if>
+    </spring:bind>
+
+    <div data-role="fieldcontain">
+    <fieldset data-role="controlgroup" id="${n}answer">
+        <div class="radio radio-success">
```

Comment:
@jonathanmtran 
I think this is a very bad idea to do a rollback to that point.

This is not because it is in a documentation of Bootstrap that we should do it. Besides it is well known that if there's something missing in Bootstrap v3.X is the way checkboxes and radios are handled in css. 
That is why I added awsome-bootstrap-checkboxes librairy (MIT License).
[https://github.com/flatlogic/awesome-bootstrap-checkbox](url)

Mostly things to see are (beyond the mere Bootstrap classes to add,):

1.  The DOM structure of the label with respect to input: The technique of implicit labels (ie without "id" and "for" attributes) raises accessibility issues. And how to deal with checkboxes and radio in bootstrap v4 (this has not changed).

2. Inline labels raise a problem of use (confusion) in a series of three or more radios. from the second radio, some people do not know if the previous label is not associated with the following radio...

3. Last, in terms of responsive layout, one can have in dashboard or mobile view (Fablet) this kind of display:
x-labelA x-labelB
x-labelC
which is pretty un-cool in terms of look and feel.

So :-1:  for inlines labels.